blocks/library/latest-posts/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Server-side rendering of the `core/latest-posts` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/latest-posts` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the post content with latest posts added.
+ */
+function gutenberg_block_core_latest_posts( $attributes ) {
+	$posts_to_show = 5;
+
+	if ( array_key_exists( 'poststoshow', $attributes ) ) {
+		$posts_to_show = $attributes['poststoshow'];
```

Comment:
Since there is no sanitization or validation of block attributes (yet), it should probably be done here, ensuring it is a number and that it is greater than 0, but less than (maybe) 100. 

See also https://github.com/WordPress/gutenberg/issues/886#issuecomment-305518238